.github/workflows/win.yml

```diff
@@ -0,0 +1,18 @@
+name: Go
+on: [push]
```

Comment:
@guerinoni maybe the windows build is not starting because the branch is coming from a pull-request (from a fork). Your PR is the first one coming from a fork as before we were only creating pull-requests from branches in the current repository (adroll).
Do you think that we should add 
```
on: [push, pull_request]
```
to make it work?
If that 's true then we should add it to all github actions workflows

Reply:
From docs
```
# Triggered when code is pushed to any branch in a repository
on: push
```

Reply:
Indeed but the source  branch of your PR comes from your forked repository.
I'm not sure about this but I think we should try since at the moment no github actions are built, whereas they always did before when PR branches were located on AdRoll repository;

Reply:
After reading the docs more thoroughly it indeed seems that we're missing `pull_request` if we want workflows to be triggered from PR coming from forked repositories.
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull-request-events-for-forked-repositories